### PR TITLE
PXC-3378 Documentation lacks clarity on upgrade from backup (8.0)

### PR DESCRIPTION
**Pull request by patrickbirch**
new file:   docs/upgrade-from-backup.md
	modified:   mkdocs-base.yml

### CODE REVIEW

File: docs/upgrade-from-backup.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Restore a 5.7 backup to an 8.0 cluster
+
+!!! important
+
+    During an upgrade or downgrade, you may encounter issues that Percona XtraBackup or Percona XtraDB Cluster are not designed to solve.
+
+For an upgrade or downgrade, use Percona XtraBackup to backup the source server data and restore the data to a target server, and then start a different version of Percona XtraDB Cluster on the restored data.
```
Comment (alina-derkach-oaza) [inline docs/upgrade-from-backup.md]: use Percona XtraBackup to backup -> back up

Comment (kamil-holubicki) [reply docs/upgrade-from-backup.md]: There is no way to downgrade. Only upgrade.

File: docs/upgrade-from-backup.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+!!! important
+
+    During an upgrade or downgrade, you may encounter issues that Percona XtraBackup or Percona XtraDB Cluster are not designed to solve.
+
+For an upgrade or downgrade, use Percona XtraBackup to backup the source server data and restore the data to a target server, and then start a different version of Percona XtraDB Cluster on the restored data. 
+
+Be aware of the following issues:
+
+* Only restore a server with a server downgrade if the source server and the target server are in the same series. Downgrading to a different server series may cause server exits and corrupt the data.
```
Comment (kamil-holubicki) [inline docs/upgrade-from-backup.md]: It is not possible to downgrade.

File: docs/upgrade-from-backup.md (hunk 1 above)
Comment (patrickbirch) [reply docs/upgrade-from-backup.md]: Rewritten

File: docs/upgrade-from-backup.md (hunk 2 above)
Comment (patrickbirch) [reply docs/upgrade-from-backup.md]: removed

**Review by kamil-holubicki — APPROVED** [review]

File: docs/upgrade-from-backup.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    !!! important
+
+        [Downgrading is not supported](https://docs.percona.com/percona-server/8.0/downgrade.html)
+
+
+## Restore a database with a different server version
+
+Review [Before you begin](https://dev.mysql.com/doc/refman//8.0/en/upgrade-before-you-begin.html).
+
+Upgrade the nodes one at a time. The primary node should be last node to be upgraded. The following steps are required on each node.
```
Comment (alina-derkach-oaza) [inline docs/upgrade-from-backup.md]: should be THE last

File: docs/upgrade-from-backup.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
+# Restore a 5.7 backup to an 8.0 cluster
+
+
+Use Percona XtraBackup to backup the source server data and restore the data to a target server, and then upgrade the server to a different version of Percona XtraDB Cluster.
```
Comment (alina-derkach-oaza) [inline docs/upgrade-from-backup.md]: Use Percona XtraBackup to BACK UP

File: docs/upgrade-from-backup.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+4. Start the database server on the target server
+
+5. Do a slow shutdown of the database server. This shutdown type flushes InnoDB operations before completing with the `SET GLOBAL innodb_fast_shutdown=0` statement. This type of shutdown may take longer.
+
+6. Install the new database server version on the target server
+
+7. Start the new database server version on the restored data directory
+
+8. Perform any other upgrade steps as necessary.
```
Comment (alina-derkach-oaza) [inline docs/upgrade-from-backup.md:28]: Put the full stops at the end of every step for consistency.

**Review by alina-derkach-oaza — APPROVED** [review]
LGTM

**Pull request merged**